capa/rules.py

```diff
@@ -277,6 +277,42 @@ def parse_description(s, value_type, description=None):
     return value, description
 
 
+def parse_meta(meta):
+    """
+    further process meta items such as MBC
+    """
+    mbcs = meta.get("mbc", [])
+    if not isinstance(mbcs, list):
+        raise InvalidRule("MBC mapping must be a list")
+
+    if mbcs:
+        meta["mbc"] = [parse_canonical_mbc(mbc) for mbc in mbcs]
```

Comment:
```suggestion
    meta["mbc"] = [parse_canonical_mbc(mbc) for mbc in mbcs]
```

Reply:
this way the key is always present and simplifies client code (don't have to check for presence)